evennia/accounts/accounts.py

```diff
@@ -301,6 +306,7 @@ def puppet_object(self, session, obj):
         # re-cache locks to make sure superuser bypass is updated
         obj.locks.cache_lock_bypass(obj)
         # final hook
+        OBJECT_PUPPET.send(sender=obj, account=self, session=session)
```

Comment:
This seems to be in a strange place - just before the `at_post_puppet` hook that may do a lot of things. It seems clearer overall to put hooks like this very last. This should then be called `OBJECT_POST_PUPPET`. Or why not name it after the hook (`AT_POST_PUPPET`)?